modules/gridBidAdapter.md

```diff
@@ -35,6 +36,19 @@ Module that connects to Grid demand source to fetch bids.
                        }
                    }
                ]
-           }
+           },
+           {
+               code: 'test-div',
+               sizes: [[728, 90]],
+               mediaTypes: { video: {} },
+               bids: [
+                   {
+                       bidder: "grid",
+                       params: {
+                           uid: 11
+                       }
+                   }
+               ]
+          }
```

Comment:
Thanks for good test parameters which return valid bid responses! 

This is great.